src/VecSim/algorithms/brute_force/brute_force_single.h

```diff
@@ -52,49 +52,36 @@ class BruteForceIndex_Single : public BruteForceIndex<DataType, DistType> {
 #endif
 protected:
     // inline definitions
+    void setVectorId(labelType label, idType id) override { labelToIdLookup.emplace(label, id); }
```

Comment:
no required changes in brute_force_multi?

Reply:
No, as we do not update vectors in multi once we get a new vector for some existing label, we append it instead